draft-ietf-webpush-encryption.md

```diff
@@ -252,15 +251,20 @@ into separate discrete steps using HMAC with SHA-256:
    auth_secret = <from User Agent>
 
    -- For both:
+   ---- HKDF Combining shared and authentication secret
    PRK_key = HMAC-SHA-256(auth_secret, ecdh_secret)
    key_info = "WebPush: info" || 0x00 || ua_public || as_public
-   IKM = HMAC-SHA-256(PRK_cek, key_info || 0x01)
+   key_L = 32
```

Comment:
Since this value isn't actually used (I've expanded HKDF into HMAC), then I'd just add a comment here.